src/main/kotlin/no/nav/familie/ef/iverksett/konsistensavstemming/KonsistensavstemmingController.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.ef.iverksett.konsistensavstemming
+
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import javax.servlet.http.HttpServletResponse
+
+@RestController
+@RequestMapping("/api/konsistensavstemming")
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class KonsistensavstemmingController(
+        private val konsistensavstemmingService: KonsistensavstemmingService,
+) {
+    @PostMapping
+    fun startKonsistensavstemming(@RequestBody konsistensavstemmingDto: KonsistensavstemmingDto, response: HttpServletResponse) {
```

Comment:
For konsistensavstemmingen trenger vi å sende alle oppdrag sine tilkjente ytelser slik det er nå, hvis ikke må vi legge til støtte for http-transaksjon